viam-cartographer/src/io/submap_painter.cc

```diff
@@ -1,138 +0,0 @@
-// This is an experimental integration of cartographer into RDK.
-#include "../src/io/submap_painter.h"
```

Comment:
Can you please provide the diff of this file & the file it was overriding in cartographer?